Swat/SwatTextareaEditor.php

```diff
@@ -407,46 +409,52 @@ function clean(elem) {
 				}
 				toRemove.forEach(r => data.node.removeChild(r));
 			},\n" .
-            "\tmenubar: false,\n" .
+        "\tmenubar: false,\n" .
+        "\tcolor_map: [\n" .
+        "\t\t'ffefee', 'Pitfall',\n" .
```

Comment:
this shouldn't be in Swat. We should somehow make these editable as an array property and then have it set in Rap or emrap.